### ISSUE TITLE
is honeywell CK65 supported?

### ISSUE THREAD

**Issue by sorosons** [luis901101/honeywell_scanner#14]
Hello There is honeywell CK65 supported on this library if not what can ı do for make app to honeywell CK65

**Comment by luis901101** [comment]
Hi, the model **CK65** appears in the **Supported devices**, so it should be supported. That list is the official list published by Honeywell. Anyway you can update the `DataCollection.aar` library with the latest from Honeywell and use the plugin as it is., check this https://github.com/luis901101/honeywell_scanner/issues/8

**Comment by novas1r1** [comment]
I'm working with a CK65 and this package. So far it works perfectly fine :)!

**Comment by novas1r1** [comment]
The last days I did have a lot of jank in my app and the scanner reacted really slow, even on simple screens. Maybe @luis901101 : can you let me know if the same happens to you when trying the CK65 with this package?

**Comment by luis901101** [comment]
> The last days I did have a lot of jank in my app and the scanner reacted really slow, even on simple screens. Maybe @luis901101 : can you let me know if the same happens to you when trying the CK65 with this package?

I don't have a CK65 to test. I recommend you to test the example project directly in your device and compare.

**Comment by novas1r1** [comment]
@luis901101 sorry, I wanted to mention @sorosons for the test and mixed it up...

I checked the example project. It has the same jank when I'm adding a TextFormField and focus it. Do you have any idea how this could be solved or who I can ask for help?

**Comment by luis901101** [comment]
@novas1r1  not sure what you mean by jank when adding TextFormField and focus it. 
Are you using DataWedge instead of native scanner implementation? Note that for DataWedge there is no meed to use this plugin.
Check you are using flutter latest version and honeywell .aar latest.
Also test your code in profile/release mode because debug mode is slow.

**Comment by sorosons** [comment]
I have try in CK65 and it works fine thanks all guys

**Comment by novas1r1** [comment]
@luis901101 I did run the application in profile mode, not debug mode. I'm not using DataWedge. I can't download the .aar file, since it doesn't work on mac. I will try to find someone who can download it for me and try it with this solution.

By jank I mean high red peaks in profile mode which lead to slow UI. Clicking buttons, any other touch gestures on the app etc are slow, it feels laggy. I did try it with the example project and added a TextFormField since I'm using forms in my application and wanted to check if it's my app or the same thing happens in the example. I guess it's more a device problem, than a plugin problem - so feel free to close this issue :)! Thanks for your help!

**Comment by luis901101** [comment]
Yeah @novas1r1 it seems more like a flutter problem or device problem, because buttons and the rest of the UI are not affected by this plugin, so laggy behavior seems related to another issue.
Yes, downloading the .aar is cumbersome, windows required 🤦🏻‍♂️.
Try app sample code in emulator, or another device no matter if it is a PDA or not, thing is to test laggy behavior, and make sure using latest flutter or different flutter versions to discard any Skia problem.
I won't close the issue now, maybe you will find some new info and can share it later.
